### ISSUE TITLE
The appeal-timeline component used in vets-website is not compliant with the latest Axe/WCAG standards

### ISSUE THREAD

**Issue by IGallupSoCo** [department-of-veterans-affairs/va.gov-team#71771]
## Issue Description
_Accessibility package upgrades being attempted by the Site Reliability Team have uncovered live a11y violations in vets-website.  The appeal-timeline component used on page workflows for showing key steps of the VA appeal process is not compliant with the axe-core 4.8 ruleset.  The rule being violated is the_ *list* _rule, which has a user impact rating of *serious*, and can affect end users with blindness disabilities who are using accessibility tools on the va.gov website._

This issue was initially discovered during local e2e testing after the Axe package updates were made, and was flagged by the following e2e test spec:
- claims-status/tests/e2e/07.appeal-status.cypress.spec.js

This issue has been verified in staging.va.gov with two versions of the Axe DevTools extensions on two different browsers.
<img width="1464" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK3RTQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--690245056d24017184ef9edb82b191ba5b854706/appeal-timelineAxeViolation.png" alt="appeal-timelineAxeViolation.png" />

More details about this specific accessibility rule can be found here: 
https://dequeuniversity.com/rules/axe/4.8/list?application=axeAPI

Due to the nature of the platform's CI workflows, the Axe package upgrades cannot be done for vets-website until the violations are resolved and the e2e test pass again.
---
## Tasks
- [ ] _The appeal-timeline component must be updated to ensure that all ordered lists contain only li content elements_

## Acceptance Criteria
- [ ] _The *list* violation is no longer detected during local e2e testing after the Axe packages have been updated as follows:_
---
New Axe package versions:
-     @axe-core/react 4.8.2
-     axe-core 4.8.2
-     cypress-axe 1.5.0

**Comment by IGallupSoCo** [comment]
src/applications/claims-status/components/appeals-v2/PastEvent.jsx
<img width="480" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd3BhQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e27d5b16120fb1431c681029be9225a68901aef2/image.png" alt="image.png" />

**Comment by IGallupSoCo** [comment]
Specifically, the *li* elements having the presentation role results in screen readers detecting an *ol* element (with no such role) that has no *li* children visible.

**Comment by micahchiang** [comment]
Hi there - checking in for the Design System team. The VA design system does not maintain a timeline component. This issue should be addressed by whichever VFS team maintains the claims status tool.
